documentation/src/main/asciidoc/topics/con_cache_encoding.adoc

```diff
@@ -0,0 +1,45 @@
+[id='encoding']
+= Cache encoding
+Encoding is the format, or media type, that {brandname} uses to store entries (key/value pairs) in caches.
+
+[discrete]
+== Remote caches
+
+{brandname} Server stores entries in remote caches without any media type information by default.
+
+Encoding configures how {brandname} stores entries in a remote cache.
```

Comment:
another solid point. I've updated this to incorporate that distinction but I've dropped the details about the different ways you can configure client encoding because I think it's TMI for this paragraph.

Reply:
TMI was on my mind as well, that's why I didn't make an outright change suggestion :)